db/migrate/20220321091220_add_personal_team_id_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPersonalTeamIdToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :personal_team_id, :integer, not_null: true
```

Comment:
```suggestion
    add_column :users, :personal_team_id, :integer, null: true
```
must be null since api user do not have a personal team